index.js

```diff
@@ -1,17 +1,20 @@
-module.exports = (robot, _, Settings = require('./lib/settings')) => {
-  robot.on('push', receive)
+const getConfig = require('probot-config')
 
-  async function receive (context) {
+module.exports = (robot, _, Settings = require('./lib/settings')) => {
+  robot.on('push', async context => {
     const payload = context.payload
     const defaultBranch = payload.ref === 'refs/heads/' + payload.repository.default_branch
 
+    const config = getConfig(context, 'settings.yml')
+    console.log(config)
```

Comment:
Can we change this to `context.log()`? Might as well get the free extra data 👌

Reply:
@JasonEtco - or we can delete it. That's a remnant of some of my debugging.